frontend/src/app/pages/exercises/exercise/shared/exercise-map/feature-managers/simulated-region-feature-manager.ts

```diff
@@ -102,6 +103,38 @@ export class SimulatedRegionFeatureManager
         elementFeature.changed();
     }
 
+    public override onFeatureDrop(
+        dropEvent: TranslateEvent,
+        droppedFeature: Feature<any>,
+        droppedOnFeature: Feature<any>
+    ) {
+        const droppedElement = this.getElementFromFeature(droppedFeature);
+        const droppedOnSimulatedRegion = this.getElementFromFeature(
+            droppedOnFeature
+        ) as SimulatedRegion;
+        if (!droppedElement || !droppedOnSimulatedRegion) {
+            console.error('Could not find element for the features');
+            return false;
+        }
+        if (
+            ['vehicle', 'personnel', 'material', 'patient'].includes(
```

Comment:
Could you please create a separate variable like so?
```ts
const dropableFeatureTypes = ['vehicle', 'personnel', 'material', 'patient']
```

Reply:
We discussed this, it is fine how it is now.

Reply:
Just FYI: if `droppedElement.type` should ever have correct typings, then 

```ts
['vehicles', 'aRandomString'].includes(droppedElement.type)
```
would be ok, even though this would probably be a mistake.
In contrast

```ts
droppedElement.type === 'vehicles' || droppedElement.type === 'aRandomString' 
```

should give type errors as `vehicles` and `aRandomString` are not assignable to `'vehicle' | 'patient' | ...`